pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -87,6 +88,20 @@ <h2 class="my-4 font-bold">2. ORIGINATOR DETAILS</h2>
             <input type="hidden" class="originator_birth_place_of_birth" value="{{ $originatorBirth.PlaceOfBirth }}" />
           </div>
         </div>
+        {{ else }}
+        <div class="grid gap-6 my-4 md:grid-cols-2">
```

Comment:
Empty fields are added here for `Date of Birth` and `Place of Birth` to allow users to provide this information since they are able to make updates in the transaction accept form.